Watchman.DomainModel/Mute/Commands/AddMuteEventToBaseCommand.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Watchman.Cqrs;
+
+namespace Watchman.DomainModel.Mute.Commands
+{
+    public class AddMuteEventToBaseCommand : ICommand
```

Comment:
te "ToBase" są bez sensu
bo z poziomu użytkownika domeny nie wiesz gdzie te dane sie zapisują 
to jest tajemnica domeny, która może zapisywać coś do bazy, ale równie dobrze może przekazywać to do jakiegoś api, kolejki, pliku, albo trzymać w pamięci